python/examples/bert/bert_rpc_client.py

```diff
@@ -0,0 +1,124 @@
+# coding:utf-8
+# pylint: disable=doc-string-missing
+import os
+import sys
+import numpy as np
+import paddlehub as hub
+import ujson
+import random
+import time
+from paddlehub.common.logger import logger
+import socket
+from paddle_serving_client import Client
+from paddle_serving_client.utils import MultiThreadRunner
+from paddle_serving_client.utils import benchmark_args
+from bert_reader import BertReader
+
+args = benchmark_args()
```

Comment:
I think this is for benchmark, not rpc client example. we should show simple example so that users can modify from your example in their application.